docker/properties-builder.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+cat > $PROP_FILE <<EOF
+#Database Name
+dbname=${HYGIEIA_API_ENV_SPRING_DATA_MONGODB_DATABASE:-dashboarddb}
+
+#Database HostName - default is localhost
+dbhost=${MONGODB_HOST:-10.0.1.1}
+
+#Database Port - default is 27017
+dbport=${MONGODB_PORT:-27017}
+
+#Database Username - default is blank
+dbusername=${HYGIEIA_API_ENV_SPRING_DATA_MONGODB_USERNAME:-dashboarduser}
+
+#Database Password - default is blank
+dbpassword=${HYGIEIA_API_ENV_SPRING_DATA_MONGODB_PASSWORD:-dbpassword}
```

Comment:
please fix the properties to match the collector

Reply:
Done